aspects/request_timeout_handler_test.go

```diff
@@ -0,0 +1,90 @@
+package aspects
+
+import (
+	"github.com/julienschmidt/httprouter"
+	"github.com/prebid/prebid-server/config"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+const reqTimeInQueueHeaderName = "X-Ngx-Request-Time"
+const reqTimeoutHeaderName = "X-Ngx-Request-Timeout"
+
+func TestQueuedRequestTimeoutWithTimeout(t *testing.T) {
+
+	rw := ExecuteAspectRequest(t, "6", "5", true)
+
+	assert.Equal(t, http.StatusRequestTimeout, rw.Code, "Http response code is incorrect, should be 408")
+	assert.Equal(t, "", string(rw.Body.Bytes()), "Body should not be present in response")
+
+}
+
+func TestQueuedRequestTimeoutNoTimeout(t *testing.T) {
+
+	rw := ExecuteAspectRequest(t, "0.9", "5", true)
+
+	assert.Equal(t, http.StatusOK, rw.Code, "Http response code is incorrect, should be 200")
+	assert.Equal(t, "Executed", string(rw.Body.Bytes()), "Body should be present in response")
+
+}
+
+func TestQueuedRequestNoHeaders(t *testing.T) {
+
+	rw := ExecuteAspectRequest(t, "", "", false)
```

Comment:
Consider adding `TestQueuedRequestSomeHeaders` to match with `TestQueuedRequestSomeHeadersIncorrect`.

Reply:
Test added